.github.rename/workflows/pr-test.yml

```diff
@@ -5,7 +5,14 @@ jobs:
     name: Run tests
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2.1.3
+      with:
+        go-version: '1.16'
+
+    - name: Check out code into the Go module directory
```

Comment:
It's actually checking out the entire source of the repo branch here, can you rename this step please